UPDATING.md

```diff
@@ -24,6 +24,8 @@ assists people when migrating to a new version.
 
 ## Next
 
+- [26328](https://github.com/apache/superset/issues/26328): Removes the deprecated Filter Box code. To be able to upgrade, you need to first migrate your deprecated filter boxes using the `superset native-filters upgrade` command available in previous versions of Superset.
```

Comment:
```suggestion
- [26328](https://github.com/apache/superset/issues/26328): Removes the deprecated filter-box visualization type code. To be able to upgrade, you need to first migrate your deprecated filter-box charts using the `superset native-filters upgrade` command available in previous versions of Superset.
```

Reply:
The `available in previous versions of Superset` is important because 4.0 will not contain the script to migrate filter boxes.